### DISCUSSION TITLE
Alias for Domain email

### DISCUSSION THREAD

**Discussion by XtremeMaC** [simple-login/app#1899] [Q&A]
Hello, 
I've a domain: abc.com that I recently got thru Google Workspace. (still debating if it was good choice or not)
default email is me@abc.com.
Since I've yet to figure out how to do a "Catch all" with Google (probably not possible, unless I pay a lot more) I signed up for Simple Login and enabled "catch all".
However, the email account associated with Simple Login is me@xyz.com and all emails I send to spam@abc.com are redirected to me@xyz.com. This wasn't my intention. 
I want all emails to be sent to my domain email. 
Did I completely misunderstand how this works? (I guess so? very possible at 2am when I set this up, lol)

Somewhat on topic, basically I want a domain name, which I currently have abc.com. don't care about a website atm. just email. I want to be able to create aliases on the fly and want emails to pooled into 1 email or based on rules. 
What's a good provider website for this? I can ditch Google Workspace in a sec. Don't mind paying. 
TIA!

**Comment by trikotret** [comment]
Just add a new mailbox you want to receive all emails. Then make it default email so everything will go there

**Reply by XtremeMaC** [reply]
It won't allow me to add my domain email.
I've tried multiple email addresses like admin@xyz.com..

**Reply by trikotret** [reply]
ohhhhhhh. Basically you want to setup abc.com domain catchall. I think the cheapest way would be open up a new free gmail account. Make that gmail as primary email domain in SL. You will get all catchall emails forwarded to that email. You can reply to your emails thru that new gmail. Make sure you reply using the reverse alias that was created from catchall
